### DISCUSSION TITLE
faiss数据库建的时候，文档3万多篇后速度太慢，有加速的办法吗

### DISCUSSION THREAD

**Discussion by wusuyan** [chatchat-space/Langchain-Chatchat#1960] [General]
向量库用的faiss，EMBEDDING_MODE 是m3e，初始化的时候3万多篇文档，要好多天，后期增加100多篇就要1个多小时，查看GPU也就使用了1.2个G左右，还有好多没有用，请问在建数据库向量化的的时候有什么办法加速吗？

**Comment by ZK8768** [comment]
我记得大文件不要用faiss

**Reply by tommy3266** [reply]
那要使用什么比较合适

**Reply by ZK8768** [reply]
我用的是pg，不过现在只是在测试，没什么文件，具体怎么样也不好说

**Comment by zRzRzRzRzRzRzR** [comment]
不要用faiss，faiss处理小数据用的
